validator/sawtooth_validator/journal/publisher.py

```diff
@@ -35,6 +37,30 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class TransactionSender(object):
```

Comment:
It is the interface implementation for transaction sending the BlockPublisher passes to the Journal/Consensus BlockPublishers. The consensus algorithms are unaware of the implementation details or how to construct one.